### ISSUE TITLE
False positive for SlotReusedDetector using it like in the Do's of the descritpion

### ISSUE THREAD

**Issue by timo-drick** [slackhq/compose-lints#424]
In version 1.4.0 and 1.4.1 i get an "Composables should only be emit from one source" error for following code that is very close to the correct implementation described in https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-component-api-guidelines.md#lifecycle-expectations-for-slot-parameters.
Also changing it to use `val movableContent = remember(content) { movableContentOf(content)}` will not remove the error detection.

```kotlin
@Composable
fun SplitLayoutVerticalSimple(
    first: @Composable () -> Unit,
    modifier: Modifier = Modifier,
    second: @Composable () -> Unit
) {
    Layout(
        modifier = modifier.clipToBounds(),
        content = {
            Box(
                Modifier
                    .layoutId("first")
                    .consumeWindowInsets(
                        WindowInsets.safeDrawing.only(WindowInsetsSides.End)
                    )
            ) {
                first()
            }
            Box(
                Modifier
                    .layoutId("second")
                    .consumeWindowInsets(
                        WindowInsets.safeDrawing.only(WindowInsetsSides.Start)
                    )
            ) {
                second()
            }
        }
    ) { measurable, constraints ->
    ...
```

**Comment by timo-drick** [comment]
Interestingly it looks like it has something todo with the AGP version. So when i use version 8.5.2 compose lint checks do not complain anything. But when i update to AGP version 8.7.0 than it starts complaining.

**Comment by timo-drick** [comment]
So a sample repo is here: https://github.com/timo-drick/compose_edge_to_edge_preview/tree/feature/testing_support
Please not currently it is in the testing_support branch.
When you remove the disabled SlotReused from lint the lint task will fail.
It is in file: https://github.com/timo-drick/compose_edge_to_edge_preview/blob/feature/testing_support/edge_to_edge_preview/build.gradle.kts
Line 47
Maybe i will change some things in the next days.
But i think it is pretty clear that this problem was introduced with AGP 8.7.0 in AGP 8.6.1 the rule does work normally i think
